docker-compose.serverless.yml

```diff
@@ -0,0 +1,273 @@
+version: '3'
+services:
+  # This extends the version in docker-compose.yml, by adding the extra depends_on for the serverless images
+  IntegrationTests:
```

Comment:
Is there any potential issue that this `IntegrationTests` service could override the `IntegrationsTests` service from `docker-compose.yml`? Maybe we could rename this to `IntegrationTests.Serverless`?

Reply:
> Is there any potential issue that this IntegrationTests service could override the IntegrationsTests service from docker-compose.yml? Or is it intentional so that this can be combined with that service?

[It's intentional, they get combined](https://docs.docker.com/compose/extends/#adding-and-overriding-configuration) 🙂

Reply:
Though I'm actually thinking it might make sense to create `IntegrationTests.Serverless` anyway, and split it into its separate section. It would probably make things simpler overall, and means more parallelism